pipelines/go/remediate.py

```
@@ -0,0 +1,539 @@
+                        "reachable fixable vulnerabilities remain at selected versions: "
+                        f"{json.dumps(details, sort_keys=True)}"
+                    )
+                for path, fixed, _ in requested:
+                    run(
+                        ["go", "mod", "edit", f"-require={path}@{fixed}"],
+                        module,
+                        gowork_off=True,
+                    )
+                reconcile(root, [module], tidy, module_pattern)
+                selected = module_versions(module)
+                for path, fixed, ids in requested:
```

Ruff (0.16.2)

[warning] Loop control variable `fixed` not used within loop body

(B007)